its/src/test/java/com/sonar/it/csharp/MultipleProjectsTest.java

```diff
@@ -74,6 +74,12 @@ public void projectTypesInfoIsLogged() {
     assertThat(buildResult.getLogs()).contains("Found 4 MSBuild C# projects: 2 MAIN projects. 2 TEST projects.");
   }
 
+  @Test
+  public void roslynVersionIsLogged() {
+    // FirstProject + FirstProjectTests + 2x SecondProject (each target framework has its log) + SecondProjectTests
+    assertThat(buildResult.getLogsLines(x -> x.startsWith("INFO: Roslyn version: "))).hasSize(5);
```

Comment:
Would it make sense to check the roslyn version too? Just to make sure it is in the correct format.

Reply:
No, as that's already tested in UT. This integration makes sure the message just goes end-to-end.